docs/multiple-connections.md

```diff
@@ -291,6 +292,20 @@ try {
         
 ```
 
+If you want to use `slave` in raw queries, you also need to explicitly specify the query runner.
```

Comment:
Why not change the above message and example  so it shows both?  Could lead to more concise documentation.

Reply:
I think it's better to keep them separate. Although the way we create `QueryRunner` was similar in both examples, what I'm really trying to show here is "how to specify QueryRunner for raw queries". Imo this saves readers' time and makes things more clear.